README.md

```diff
@@ -13,6 +15,22 @@ to generate systemd service configuration to run a container.
 
 ## How to contribute
 
+### Testing
+
+RPM packages if Hirte project are available on
+[hirte-snapshot](https://copr.fedorainfracloud.org/coprs/mperina/hirte-snapshot/) COPR repo. To install hirte packages
+on your system please add that repo using:
+
+```
+dnf copr enable mperina/hirte-snapshot centos-stream-9
```

Comment:
We should build for more than just CS9, Fedora as well

Reply:
Fixed